site/content/en/docs/handbook/config.md

```diff
@@ -124,14 +124,42 @@ Some features can only be accessed by minikube specific environment variables, h
 
 ### Example: Disabling emoji
 
+{{% tabs %}}
+
+{{% linuxtab %}}
+
+```shell
+export MINIKUBE_IN_STYLE=false
+minikube start
+```
+
+{{% /linuxtab %}}
+
+{{% mactab %}}
+
 ```shell
 export MINIKUBE_IN_STYLE=false
 minikube start
 ```
 
+{{% /mactab %}}
+
+{{% windowstab %}}
+
+```shell
+$env:MINIKUBE_IN_STYLE=false
+minikube start
+```
+
+{{% /windowstab %}}
+{{% /tabs %}}
+
 ### Making environment values persistent
 
 To make the exported variables persistent across reboots:
 
 * Linux and macOS: Add these declarations to `~/.bashrc` or wherever your shells environment variables are stored.
-* Windows: Add these declarations via [system settings](https://support.microsoft.com/en-au/help/310519/how-to-manage-environment-variables-in-windows-xp) or using [setx](https://stackoverflow.com/questions/5898131/set-a-persistent-environment-variable-from-cmd-exe)
+* Windows: Either add these declarations to your `~\Documents\WindowsPowerShell\Microsoft.PowerShell_profile.ps1` or run the following in a powershell terminal:
```

Comment:
```suggestion
* Windows: Either add these declarations to your `~\Documents\WindowsPowerShell\Microsoft.PowerShell_profile.ps1` or run the following in a PowerShell terminal:
```

Reply:
TiL that it's actually PowerShell and not Powershell lol.